.github/workflows/build-release.yml

```diff
@@ -98,103 +105,15 @@ jobs:
           cd release/
           7z.exe a ../${{ matrix.archive_name }} rage/
         shell: bash
-        if: matrix.name == 'windows'
+        if: (matrix.name == 'windows' && startsWith(steps.get_version.outputs.VERSION ,'v'))
 
-      - name: Get the version
-        id: get_version
-        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
       - name: Upload archive to release
         uses: svenstaro/upload-release-action@v1-release
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: ${{ matrix.archive_name }}
           asset_name: rage-${{ steps.get_version.outputs.VERSION }}-${{ matrix.asset_suffix }}
           tag: ${{ github.ref }}
+        if: (startsWith(steps.get_version.outputs.VERSION ,'v'))
 
-  deb:
-    name: Debian ${{ matrix.name }}
```

Comment:
I definitely want the Debian release process to be tested as part of #93. There are a number of complex parts to the Debian packaging configuration that I have broken before prior releases 😅